File: OrientacaoAObjetos/JavaIntro/src/Aula01/Ex01.java
package Aula01;

import java.util.Scanner;

public class Ex01 {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        double salarioBase;
        double salarioLiquido;

            System.out.print("Digite seu salario-base: ");
                salarioBase = scanner.nextDouble();
            salarioLiquido = salarioBase + (salarioBase * 0.5) - (salarioBase * 0.7);

        System.out.println(salarioLiquido);
        scanner.close();
    }
}

File: orientacaoAObjetos/Python/Aula01/ex01.py
salario_base = float(input("Digite o o seu salario base: "))
gratificação = (salario_base*5) / 100
imposto = (salario_base * 7) / 100

salario_final = (salario_base + gratificação) - imposto

print("seu salario final é de: ", salario_final)